config.py

```diff
@@ -12,14 +12,17 @@
 
 
 def fetch_resource(rsrc_path):
-  """Loads resources from the temp dir used by pyinstaller executables"""
-  try:
-    base_path = Path(sys._MEIPASS)
-  except AttributeError:
-    return rsrc_path  # not running as exe, just return the unaltered path
-  else:
-    return base_path.joinpath(rsrc_path)
+    """Loads resources from the temp dir used by pyinstaller executables"""
+    if hasattr(sys, '_MEIPASS'):
+        base_path = Path(getattr(sys, '_MEIPASS'))
+        return base_path.joinpath(rsrc_path)
+    else:
+        # not running as exe, just return the unaltered path
+        return Path(rsrc_path)
 
-# DB = Database(Path(os.path.abspath("database.csv")))
-DB = Database(fetch_resource(r"database/database.csv"))
-userSession = EmployeeSelf(Employee(Permission_level=1))
\ No newline at end of file
+
+if getattr(sys, 'frozen', False) and hasattr(sys, '_MEIPASS'):
+    DB = Database(fetch_resource(r"database/database.csv"))
+else:
+    DB = Database(Path(os.path.abspath("database.csv")))
```

Comment:
How about we just put the database csv in a folder called database, because there are other areas that reference this path. (I know profile for sure uses it on the update button) That way we don't need to deal with two different paths